requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy
 xarray
 pysnptools>=0.4.19
--e git://github.com/pystatgen/sgkit.git#egg=sgkit
+git+https://github.com/pystatgen/sgkit
```

Comment:
I changed this because I was getting an import error since pytest was processing the tests for sgkit itself (even though it didn't try to run them), and sgkit tests require some packages that sgkit-plink doesn't use. Removing the `-e` option means that the source isn't installed, and that avoids this error.

For dev we might want the `-e`, but that can be overridden manually - and just when it is needed.

Reply:
Makes sense, good to know.